bedrock-backup/get-bedrock-pod-name.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 
 release=$1
-kubectl get pod -l "app.kubernetes.io/instance=${release}" -o json | jq -r .items[0].metadata.name
+kubectl get pod -l "app.kubernetes.io/instance=${release}" -o json | jq -r '.items[0].metadata.name // empty'
```

Comment:
Not sure I get this one?

Reply:
if `release` value is empty, or release doesn't exist, the json will be empty and `jq` call will return `null` value. The alternative operator `//` tells `jq` to return an empty sting in this case, so the script won't report back "null" as the pod name, which simplifies check on the caller's side.

Reply:
Hmmm....is it really easier to check for an empty string than null?

Reply:
I think in bash it is a bit easier: `[[ -z "${var}" ]]` vs `[[ "${var}" == "null" ]]`, [this line](https://github.com/hansenms/minecraft-server/pull/9/files#diff-352a1d479c074600340ca6cfe778ad4ddd1a7b5b705e09f803298a7202589209R16) in the script.
Not critical for me tho, can revert these parts.

Reply:
Nah, probably fine. Just some of the jq syntax that is a bit opaque to me.